tests/e2e/scale/upgrade/test_upgrade_with_scaled_rgw_obc.py

```diff
@@ -0,0 +1,186 @@
+import logging
+import pytest
+import time
+import os
+
+from ocs_ci.ocs import constants, scale_noobaa_lib
+from ocs_ci.ocs.resources.objectconfigfile import ObjectConfFile
+from ocs_ci.framework.pytest_customization.marks import (
+    pre_upgrade,
+    post_upgrade,
+    skipif_managed_service,
+    skipif_bm,
+    skipif_external_mode,
+    vsphere_platform_required,
+)
+from ocs_ci.utility.utils import ocsci_log_path
+from ocs_ci.utility import utils, templating
+from ocs_ci.ocs.exceptions import UnexpectedBehaviour
+
+log = logging.getLogger(__name__)
+
+# Namespace and noobaa storage class
+namespace = constants.OPENSHIFT_STORAGE_NAMESPACE
+sc_name = constants.DEFAULT_STORAGECLASS_RGW
+# Number of scaled obc count
+scale_obc_count = 100
+# Number of obc creating by batch
+num_obc_batch = 50
+# Number of objects
+num_objs = 150000
+# Scale data file
+log_path = ocsci_log_path()
+obc_scaled_data_file = f"{log_path}/obc_scale_data_file.yaml"
+
+
+@pre_upgrade
+@vsphere_platform_required
+@skipif_external_mode
```

Comment:
Not sure, but this test should work on ext mode as well right?

Reply:
Do you know who can confirm this?